tests/data/simpler_model_data.json

```diff
@@ -0,0 +1 @@
+[{"basal_dendrite": [0.8181731458339566, 0.7744795652021181, 0.7421541450112694, 0.6520072193791582, 0.6485882473330047, 0.1349793202635492, 0.13281142062730328, 0.04714861655668604, 0.8068017690335927, 0.10092123989396076, 0.04680877854080963, 0.03134206997162865, 0.7827281142242415, 1.0, 0.7721448184675089, 0.9855848859679315, 0.9534542307119734, 0.6962178219191657, 0.7640790516447591, 0.8660723409534996, 0.032019353948400074, 0.855147949386722], "apical_dendrite": [1.0, 0.7443870208169504, 0.3647110695224464, 0.12184497718515643, 0.3516306644940961, 0.7126552920188522, 0.017892523333600956, 0.006267798332285662, 0.8881232630075685]}, {"basal_dendrite": [3.0850000381469727, 0.9300000071525574, 0.6642856597900391, 0.4165624976158142, 0.31000006198883057, 0.9300000071525574, 0.7045454382896423, 0.7233332991600037, 0.7233333587646484, 1.1119999885559082, 0.7733333110809326, 0.9300000071525574, 0.4871428310871124, 1.2333332300186157, 0.3752632141113281, 0.8524999618530273, 0.6842105388641357, 0.6197144389152527, 0.15000000596046448, 0.9300000071525574, 0.9300000071525574, 0.33268293738365173], "apical_dendrite": [2.680000066757202, 1.7899999618530273, 2.0899999141693115, 1.6699998378753662, 0.32999998331069946, 0.5162963271141052, 0.30000001192092896, 0.6000000238418579, 0.3656250238418579]}, {"basal_dendrite": 6.981783576001451, "apical_dendrite": 5.776746786628541}]
```

Comment:
Don't forget to add new lines at the end of each file

Reply:
ah why?

Reply:
done